x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_forms/events/linux.tsx

```diff
@@ -96,13 +92,16 @@ export const LinuxEvents = React.memo(() => {
       type={i18n.translate('xpack.endpoint.policy.details.eventCollection', {
         defaultMessage: 'Event Collection',
       })}
-      supportedOss={useMemo(
-        () => [i18n.translate('xpack.endpoint.policy.details.linux', { defaultMessage: 'Linux' })],
-        []
-      )}
-      id="linuxEventsForm"
+      description={i18n.translate('xpack.endpoint.policy.details.eventCollectionLabel', {
+        defaultMessage: 'Event Collection',
+      })}
+      supportedOss={i18n.translate('xpack.endpoint.policy.details.linux', {
```

Comment:
why did you get rid of useMemo here again?

Reply:
`supportedOss` was taking an array of translated strings and then concat'ing them with ', '. Since we do i18n, the translator needs to be able to specify that text. `supportedOss` now takes a React node. Since `i18n.translate` is returning a string, there is no need to memoize.